src/coprocessor/tracker.rs

```diff
@@ -136,18 +137,16 @@ impl Tracker {
     /// TiDB asks for ExecDetail to be printed in its log.
     pub fn get_item_exec_details(&self) -> kvrpcpb::ExecDetails {
         assert!(self.current_stage == TrackerState::ItemFinished);
-        let is_slow_query = time::duration_to_sec(self.item_process_time) > SLOW_QUERY_LOWER_BOUND;
         let mut exec_details = kvrpcpb::ExecDetails::new();
-        if self.req_ctx.context.get_handle_time() || is_slow_query {
-            let mut handle = kvrpcpb::HandleTime::new();
-            handle.set_process_ms((time::duration_to_sec(self.item_process_time) * 1000.0) as i64);
-            handle.set_wait_ms((time::duration_to_sec(self.wait_time) * 1000.0) as i64);
-            exec_details.set_handle_time(handle);
-        }
-        if self.req_ctx.context.get_scan_detail() || is_slow_query {
-            let detail = self.total_exec_metrics.cf_stats.scan_detail();
-            exec_details.set_scan_detail(detail);
-        }
+
+        let mut handle = kvrpcpb::HandleTime::new();
+        handle.set_process_ms((time::duration_to_sec(self.item_process_time) * 1000.0) as i64);
+        handle.set_wait_ms((time::duration_to_sec(self.wait_time) * 1000.0) as i64);
+        exec_details.set_handle_time(handle);
```

Comment:
This behavior is more frequently than before, does it impact the performance?

Reply:
We always record these durations actually, so the cost in this change is only setting the structure, which should be very small.